### DISCUSSION TITLE
How to install in android

### DISCUSSION THREAD

**Discussion by jroe1943** [artyom-beilis/skyhopper#27] [General]
I can't find the '...' to download AstroHopper for no internet connection?

**Comment by jroe1943** [comment]
I found it!  Seems like in one of my phones it was in full screen mode and I couldn't see the downloads link.

**Reply by artyom-beilis** [reply]
Yes it should be in your browser to install it.

Probably some pictures need to be added to the tutorial

**Reply by bird0cosmic** [reply]
Why should you not use astrohopper on a laptop?

**Reply by artyom-beilis** [reply]
> Why should you not use astrohopper on a laptop?

Because having motion sensors are essential for its functionality. Most laptops don't have ones.
